src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -454,6 +454,7 @@ export function DistributionForm({
                   )
                 )} ${giftVaultSymbol}`}
                 onChange={value => {
+                  if (value == '') value = '0';
```

Comment:
@topocount I needed to add this to fix the error I was encountering. Not sure if this is the best approach to this, but seems to work for now.